web/packages/teleport/src/Users/Users.tsx

```diff
@@ -49,16 +49,31 @@ export function Users(props: State) {
     onUpdate,
     onDelete,
     onReset,
+    onStartInviteCollaborators,
+    onInviteCollaboratorsClose,
+    inviteCollaboratorsOpen,
   } = props;
-
   return (
     <FeatureBox>
       <FeatureHeader>
         <FeatureHeaderTitle>Users</FeatureHeaderTitle>
         {attempt.isSuccess && (
-          <ButtonPrimary ml="auto" width="240px" onClick={onStartCreate}>
-            Create New User
-          </ButtonPrimary>
+          <>
+            {!props.inviteCollaborators && (
+              <ButtonPrimary ml="auto" width="240px" onClick={onStartCreate}>
+                Create New User
+              </ButtonPrimary>
+            )}
+            {props.inviteCollaborators && (
```

Comment:
If I'm reading correctly, this will display `Enroll Users` for any enterprise customers since we are always sending this on enterprise features (https://github.com/gravitational/teleport.e/blob/f32252614746014c5a9136cd34a9bf3ee2dc5d8f/web/teleport/src/features.tsx#L376)

You could use the `isCloud` flag on `web/packages/teleport/src/config.ts` to show/hide this instead.

Reply:
Yup, that did the trick, thanks!